.github/workflows/main.yml

```diff
@@ -116,16 +122,16 @@ jobs:
         with:
           go-version: '${{ env.GO_VERSION }}'
 
-      - name: fetch and rebase on ${{ github.base_ref }}
+      - name: fetch and rebase on ${{ env.BASE_BRANCH }}
```

Comment:
I don't think this check is particularly useful to be running for anything that isn't a PR. Should we instead just skip the check if we're not in a PR?

Reply:
yeah, very true 😂  ok gimme a few mins